src/tileset.rs

```diff
@@ -113,10 +136,23 @@ impl Tileset {
     pub(crate) fn parse_xml_in_map(
         parser: &mut impl Iterator<Item = XmlEventResult>,
         attrs: Vec<OwnedAttribute>,
-        map_path: &Path,
+        map_path: &Path, // Template or Map file
```

Comment:
Uuuuuuhm...

Reply:
Whoops! What should I call it? The paths are tracked in multiple places and now it can be a map, template or (I think) a tilesheet.

Reply:
The name `root_path` has appeared elsewhere, and I guess it would be appropriate here as well. However, I don't really agree with the term "root", and would also be fine with just `path` (and would assume this path does not include the file name, actually).

Reply:
The function should be renamed to something that doesn't specify maps (e.g. parse_xml_in_file) and the path parameter couls just be file_path then.